hermes/management/commands/hopingest.py

```diff
@@ -1,444 +0,0 @@
-from  datetime import datetime, timezone
-import logging
-import os
-
-from django.core.management.base import BaseCommand, CommandError
-#from django.conf import settings
-
-from hop import Stream
-from hop.auth import Auth
-from hop.io import StartPosition, Metadata, list_topics
-from hop.models import GCNCircular, JSONBlob
-
-from confluent_kafka import cimpl
-
-from hermes.brokers import hopskotch
-from hermes.models import Message
-
-logger = logging.getLogger(__name__)
-#logger.setLevel(logging.DEBUG)
-
-class UpdateTopicsException(Exception):
-    pass
-
-SCIMMA_KAFKA_URL = 'kafka://kafka.scimma.org/'
-
-class Command(BaseCommand):
```

Comment:
Replacing this management command with `tom-alertstreams` `readstreams` will propagate in to the `helm-chart/templates/deployment.yaml` where we start the management command in it's container. I'm happy to make that small change when the time comes.

Reply:
Okay, I've updated the helm charts command to be readstreams. We will also need to add the scimma user/pass and gcn user/pass env variables to the k8s secrets for hermes

Reply:
I'll take care of this. 

And just a note to myself.:
* our current `HERMES_INGEST_USERNAME` should become `SCIMMA_AUTH_USERNAME`, and
* our current `HERMES_INGEST_PASSWORD` should become `SCIMMA_AUTH_PASSWORD`
* And we need to add `GCN_CLASSIC_CLIENT_ID` and `GCN_CLASSIC_CLIENT_SECRET`